Blog Post: Grant for Tor (#316)
OpenSats/website: data/blog/tor-receives-support-grant.mdx

---
title: "Strengthening Bitcoin’s Privacy with Tor"

> date: '2024-08-26'

Reviewer: ```suggestion
date: '2024-08-28'
```

---

OpenSats is pleased to announce a support grant for [The Tor
Project](https://gitlab.torproject.org/tpo/team), a nonprofit organization
dedicated to privacy and internet freedom since 2006.

Our [mission](/mission) is to provide a sustainable ecosystem of funding for
free and open-source contributors and projects, especially those that help
Bitcoin flourish. It's obvious to us that Tor is such a project.

Tor originated in the mid-1990s as a U.S. Naval Research Laboratory project to
create a secure and anonymized communications network using onion routing.
[Open-sourced in 2002](https://www.torproject.org/about/history/), this
technology led to the formation of The Tor Project, which launched the
privacy-focused [Tor Browser](https://www.torproject.org/download/) in 2008. The
Tor Project also developed tools like [Nyx](https://nyx.torproject.org/),
[Stem](https://stem.torproject.org/), and
[TorBirdy](https://gitlab.torproject.org/legacy/trac/-/wikis/torbirdy) to
enhance usability and privacy further.

The integration of the Tor network with Bitcoin has [always played a crucial
role](https://satoshi.nakamotoinstitute.org/posts/bitcointalk/45/) in

> safeguarding the ecosystem by increasing privacy:

Reviewer: ```suggestion
safeguarding the ecosystem by increasing privacy. It allows bitcoiners to:
```

> - Anonymize Network Communications: Protect miner and node operators' IP

Reviewer: ```suggestion
- **Anonymize Network Communications:** Protect IP addresses of miners, node operators, and users 
```

>   addresses from tracing and surveillance.

Reviewer: ```suggestion
  from tracing and surveillance.
```

> - Run Bitcoin Infrastructure as Onion Services: Operate nodes and other

Reviewer: ```suggestion
- [Run Bitcoin Infrastructure as Onion Services:](https://os-website-git-support-for-tor-open-sats.vercel.app/blog/tor-receives-support-grant) Operate nodes and other
```

  infrastructure as Tor [onion
  services](https://en.bitcoin.it/wiki/Setting_up_a_Tor_hidden_service) with
  `.onion` addresses, simplifying connectivity by reducing the need for
  [NAT](https://en.wikipedia.org/wiki/Network_address_translation) or firewall
  setups and allowing controlled access without relying on DNS.

> We believe everyone should be able to explore the internet with privacy.
> [...] We advance human rights and defend your privacy online through free
> software and open networks.
>
><cite>—The Tor Project</cite>

This grant will allow The Tor Project to expand its team of [onion
service](https://gitlab.torproject.org/tpo/onion-services) developers, which
will be instrumental in releasing substantial improvements to onion services,

> including a protection mechanism against [DDoS

Reviewer: ```suggestion
including an improved protection mechanism against [DDoS
```
